dashboard/src/utils/dateFormatter.js

```diff
@@ -1,5 +1,3 @@
-import moment from "moment"
-
 export const formatDate=(date)=>{
-    return moment(date).format("YYYY-MM-DD")
+    return date.toISOString().slice(0,10)
```

Comment:
Are you sure you want to use `slice()` instead of `substring()`?  (It's not a big deal, but you should give it due consideration.)